examples/colourpicker-histogram.py

```diff
@@ -0,0 +1,48 @@
+from dash import Dash, dcc, html, Input, Output
+import plotly.graph_objects as go
+import plotly.express as px
+import pandas as pd
+import dash_daq as daq
+import plotly.io as pio
+
+df = px.data.gapminder()
+df = df.loc[df['year']==1952]
+
+pio.templates.default = "ggplot2"
```

Comment:
I agree with Ann. Let's a add her comment above this templates line of code so people know what this does.
You can also add a second comment:

> Here are all the default templates:  "ggplot2", "seaborn", "simple_white", "plotly", "plotly_white", "plotly_dark", "presentation", "xgridoff", "ygridoff", "gridon", "none",